### ISSUE TITLE
Adding DirectoryNamedWebpackPlugin

### ISSUE THREAD

**Issue by loelsonk** [reactql/kit#128]
Hi @leebenson,

are there any plans to integrate [DirectoryNamedWebpackPlugin](https://github.com/shaketbaby/directory-named-webpack-plugin) into reactQL?

[Configuration](https://webpack.js.org/configuration/resolve/#resolve-plugins) should be rather simple, but I am having problems. Webpack compiles with errors.

Could you give me a hint how to get it to work?

`kit/webpack/common.js`
```javascript
...
import DirectoryNamedWebpackPlugin from 'directory-named-webpack-plugin';


// Export a new 'base' config, which we can extend/merge from
export default new WebpackConfig().merge({

  // Format the output stats to avoid too much noise
  stats,

  // Javascript file extensions that webpack will resolve
  resolve: {
    // I tend to use .js exclusively, but .jsx is also allowed
    extensions: ['.js', '.jsx'],
    plugins: [new DirectoryNamedWebpackPlugin(true)],
...
```
`localhost:8080` console output

![](https://i.imgur.com/IllgC7g.png)

`localhost:8081` console output

![](https://i.imgur.com/BZnQTeU.png)

![](https://i.imgur.com/RvRbEff.png)

**Comment by dmitry** [comment]
@loelsonk how is this can be helpful?

**Comment by leebenson** [comment]
Apologies for the generic response, but I've released a new [v3.0 kit](https://github.com/leebenson/reactql/releases/tag/3.0.0), which has been re-written from the ground up.

It bumps the ReactQL stack to Webpack 4, React 16.4, Apollo Client 2.0, and adds Styled Components, full SSR (even in development), and has been re-written in Typescript. It's leaner, faster and has been brought completely up-to-date.

If you're still using ReactQL (and I appreciate you may not be, given the time lag in my response), I'd recommend checking out the new kit to see if your issue has been answered. If not, please feel free to post a new issue at https://github.com/leebenson/reactql
